rholang/examples/linking/v0.2/packages/RhoClass.rho

```diff
@@ -0,0 +1,13 @@
+export RhoClass in {
+  contract @[*RhoClass, "apply"](name, @args) = {
+    match args {
+      [a]                => { name!(a) }
+      [a, b]             => { name!(a, b) }
+      [a, b, c]          => { name!(a, b, c) }
+      [a, b, c, d]       => { name!(a, b, c, d) }
+      [a, b, c, d, e]    => { name!(a, b, c, d, e) }
+      [a, b, c, d, e, f] => { name!(a, b, c, d, e, f) }
+      _                  => { name!(args) }
```

Comment:
Won't this invoke the same code as `name!(a)`?

Reply:
Sure, but what gets sent on name has arity 1, so if name dispatches based on the message arity as follows, the first one will get called: 
```
contract foo(a) { ... }
contract foo(a,b) { ... }
...
contract foo(a,b,c,d,e,f) { ... }
```
It doesn't look like any of your methods dispatch based on length, though, so it's a moot point.